betterdiscordctl

```diff
@@ -430,161 +317,70 @@ bdc_find_modules() {
-  flatpak_version=$("$flatpak_bin" --version | sed -n 's/Flatpak //p')
-  if [[ $(Semver::compare "$flatpak_version" '1.0.0') -eq -1 ]]; then
-    die 'ERROR: You are using an unsupported version of Flatpak.' \
-        'See https://github.com/bb010g/betterdiscordctl/issues/45'
-  fi
-  # flatpak sucks and doesn't use stderr for warnings.
-  # https://github.com/flatpak/flatpak/blob/13e449b/app/flatpak-main.c#L259-L286
-  # This really should be better for directories with newlines, but...
-  # We're just going to grab the last line and hope for the best.
-  flatpak_location=$("$flatpak_bin" info --show-location com.discordapp.Discord)
-  flatpak_location=${flatpak_location##*$'\n'}
-  if [[ -d ${flatpak_location:?}/files/discord ]]; then
-    discord=$flatpak_location/files/discord
-  else
-    discord=$flatpak_location/files/extra
-  fi
-  verbose 2 "VV: Checking $discord"
-  if [[ -d $discord ]]; then
-    verbose 1 "V: Using Discord at $discord"
-    # shellcheck disable=SC2016
-    # Expansion should happen inside flatpak's shell.
-    flatpak_config=$("$flatpak_bin" run --command=sh com.discordapp.Discord -c $'printf -- \'%s\n\' "$XDG_CONFIG_HOME"')
-    discord_config=${flatpak_config:-$HOME/.var/app/com.discordapp.Discord/config}/discord
-    if [[ ! -d $discord_config ]]; then
-      printf 'WARN: Config directory not found for Discord (%s, %s).\n' "$discord" "$discord_config" >&2
-    fi
-    bdc_find_modules
-  else
-    die 'ERROR: Discord installation not found.'
-  fi
-}
-
-bdc_nix() {
-  for flavor in "${flavors[@]}"; do
-    verbose 2 "VV: Trying flavor '$flavor'"
-    if [[ ${flavor,,} == ptb ]]; then
-      flavor=PTB
-    elif [[ ${flavor,,} == canary ]]; then
-      flavor=Canary
-    fi
-    scandir=$("$nix_bin" -r "$(which "Discord$flavor")")
-    verbose 2 "VV: Scanning $scandir"
-    discord="$scandir/opt/Discord$flavor"
-    if [[ -d $discord ]]; then
-      verbose 1 "V: Using Discord at $discord"
-      discord_config=$xdg_config/discord${flavor,,}
-      if [[ ! -d $discord_config ]]; then
-        printf 'WARN: Config directory not found for Discord %s (%s, %s).\n' \
-          "$flavor" "$discord" "$discord_config" >&2
-        continue
-      fi
-      if [[ -z $modules ]]; then
-        bdc_find_modules
-      else
-        # --modules
-        [[ -d $modules ]] || die 'ERROR: Discord modules directory not found.'
-      fi
-      break
-    fi
-  done
+  # shellcheck disable=SC2016
+  # Expansion should happen inside flatpak's shell.
+  xdg_config=$("$flatpak_bin" run --command=sh com.discordapp.Discord -c $'printf -- \'%s\n\' "$XDG_CONFIG_HOME"')
+  discord_config=${xdg_config:-$HOME/.var/app/com.discordapp.Discord/config}/discord
+  bdc_find_modules
 }
 
 bdc_kill() {
-  declare process_name=Discord
-  [[ $flavor ]] && process_name+=" $flavor"
-  printf 'Killing %s processes...\n' "$process_name" >&2
+  declare process_name="Discord $flavor"
+  printf 'Killing %s processes...\n' "${process_name% }" >&2
```

Comment:
Since we don't seem to care about the potential extra space in other messages:
```suggestion
  printf 'Killing Discord %s processes...\n' "$flavor" >&2
```